<?xml version="1.0"?>
<launch>

	<node name="acados_traj"
				pkg="rqt_plot"
				type="rqt_plot"
				args="/crazyflie/acados_traj/vector"
	/>

	<node name="cf_position"
				pkg="rqt_plot"
				type="rqt_plot"
				args="/crazyflie/external_position/point"
	/>

	<node name="cf_euler_angles"
				pkg="rqt_plot"
				type="rqt_plot"
				args="/crazyflie/euler_angles/vector"
	/>

	<node name="cf_quaternion"
				pkg="rqt_plot"
				type="rqt_plot"
				args="/crazyflie/quat"
	/>

	<node name="cf_linear_velo"
				pkg="rqt_plot"
				type="rqt_plot"
				args="/crazyflie/linear_velo"
	/>

	<node name="cf_angular_velo"
				pkg="rqt_plot"
				type="rqt_plot"
				args="/crazyflie/angular_velo"
	/>

	<node name="recording"
        pkg="rosbag"
        type="play"
        args="/home/barbara/crazyflie.bag"
	/>

</launch>
